TypeScript-tests.md

```diff
@@ -15,7 +15,7 @@ You can run the TypeScript tests like this:
 
 Python:
 ```console
-python -m scripts.typescript_checks
+python -m scripts.run_typescript_checks --strict_check
```

Comment:
Are you sure this is correct @HardikGoyal2003? I don't see a strict_check flag in the codebase.

Reply:
@seanlip  you can check this file `` scripts/pre_push_hook.py`` line 75. Thanks!

Reply:
@seanlip Typo has been corrected. Thanks!